### DISCUSSION TITLE
Hello author, please ask which version can use YOLOV5 6.1 or 6.2 to do detection to achieve the purpose of tracking

### DISCUSSION THREAD

**Discussion by JMChenl** [mikel-brostrom/yolo_tracking#493] [Q&A]
Hello author, please ask which version can use YOLOV5 6.1 or 6.2 to do detection to achieve the purpose of tracking

**Comment by mikel-brostrom** [comment]
The only Yolo version that is 100% supported is the one that is provided by this repo at the moment of cloning
